angular/package.json

```diff
@@ -51,16 +51,16 @@
     "@angular/router": "latest",
     "@ionic/core": "latest",
     "chalk": "^2.3.2",
-    "execa": "^0.9.0",
+    "execa": "^0.10.0",
     "fs-extra": "^5.0.0",
     "glob": "7.1.2",
     "inquirer": "^5.1.0",
     "listr": "^0.13.0",
-    "rxjs": "^5.5.2",
+    "rxjs": "^5.5.8",
     "semver": "^5.5.0",
     "tslint": "^5.8.0",
-    "tslint-ionic-rules": "0.0.13",
-    "typescript": "^2.6.2",
-    "zone.js": "^0.8.18"
+    "tslint-ionic-rules": "0.0.14",
+    "typescript": "2.6.2",
```

Comment:
We actually need 2.7.2, otherwise there is typing issues around "CustomEvent", can you also update it?

Thanks!

Reply:
Sure!